mk/extbld/arch-embox-clang

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
I'll be fine with variable passed. The first line can be used in new script, the compiler invocation should call clang or cross-gcc depending on $0